### ISSUE TITLE
NamedTuple basic usage: False Positive (too-few-public-methods)

### ISSUE THREAD

**Issue by jhbuhrman** [PyCQA/pylint#2459]
`NamedTuple` derived types frequently don't have any additional methods defined.

### Steps to reproduce
1. Run pylint (missing-docstring disabled) over this file (named named_tuple_pylint_check.py, find 
    attached, remove the .txt suffix - [named_tuple_pylint_check.py.txt](https://github.com/PyCQA/pylint/files/2337851/named_tuple_pylint_check.py.txt)):
    ```python
    # python3.7
    from typing import NamedTuple
    
    
    class Example(NamedTuple):
        some_int: int
    
    
    def run() -> None:
        example = Example(1)
        print(f'result: {example!r}')
    
    
    run()  # python happy, pylint error
    ```

### Current behavior
```
$ pylint --disable=missing-docstring named_tuple_pylint_check.py
************* Module named_tuple_pylint_check
named_tuple_pylint_check.py:5:0: R0903: Too few public methods (0/2) (too-few-public-methods)

------------------------------------------------------------------
Your code has been rated at 8.57/10 (...)
```

### Expected behavior
```
$ pylint --disable=missing-docstring named_tuple_pylint_check.py
Your code has been rated at 10.00/10 (...)
```

### pylint --version output
```
pylint 2.1.1
astroid 2.0.4
Python 3.7.0 (default, Jun 29 2018, 20:13:13) 
[Clang 9.1.0 (clang-902.0.39.2)]
```

**Comment by PCManticore** [comment]
Thanks for creating an issue!
